<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editExperimentDockWidget</class>
 <widget class="QDockWidget" name="editExperimentDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="stepUSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>А</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="regASpinBox">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Шаг по параметру</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="Line" name="line">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>delta</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QDoubleSpinBox" name="savingSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.200000000000000</double>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Капитал</string>
         </property>
        </widget>
       </item>
       <item row="23" column="1">
        <widget class="QSpinBox" name="capitalSpinBox">
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="26" column="1">
        <widget class="QDoubleSpinBox" name="aSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="26" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>а</string>
         </property>
        </widget>
       </item>
       <item row="27" column="1">
        <widget class="QDoubleSpinBox" name="bSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="28" column="1">
        <widget class="QDoubleSpinBox" name="xSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.300000000000000</double>
         </property>
        </widget>
       </item>
       <item row="28" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>p1</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="regP1SpinBox">
         <property name="suffix">
          <string/>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>p2</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="regP2SpinBox">
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>saving</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QDoubleSpinBox" name="deltaSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.950000000000000</double>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QSpinBox" name="exportSpinBox">
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>экспортная цена</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QSpinBox" name="woodProductionSpinBox">
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Объем заготовки</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Цена преобразований</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QDoubleSpinBox" name="CostSpinBox">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>500.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="24" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>ВВП отрасли</string>
         </property>
        </widget>
       </item>
       <item row="24" column="1">
        <widget class="QSpinBox" name="ProductionSpinBox">
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="27" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>b</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Число этапов симуляции</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="nEpochsSpinBox">
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="39" column="0">
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>editExperimentDockWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClicked()</slot>
 </slots>
</ui>
